app/policies/comment_policy.rb

```diff
@@ -1,3 +1,24 @@
-class CommentPolicy
+class CommentPolicy < ApplicationPolicy
+
+  #methods: create (always true); edit, update, and delete (only the author)
+  def edit?
+    user == record.author
+  end
+
+  def update?
+    edit?
+  end
+
+  def create?
+    true
```

Comment:
This would allow any user to create a comment on any photo. Who should be able to create a new comment?